tests/scripts/thread-cert/mesh_cop.py

```diff
@@ -30,10 +30,12 @@
 from binascii import hexlify
 from enum import IntEnum
 import io
+import logging
 import struct
 
 from network_data import SubTlvsFactory
 import common
+from tlvs_parsing import UnknownTlvFactory
```

Comment:
put this line after `from network_data import SubTlvsFactory` to keep headers order?

Reply:
accepted.